test/app.spec.js

```diff
@@ -0,0 +1,216 @@
+'use strict'
+
+const request = require('supertest')
+const chai = require('chai')
+const expect = chai.expect
+const mongoose = require('mongoose')
+const app = require('../src/app')
+
+describe('app', () => {
+  let id
+  let createdAt
+
+  const athlete = {
+    lastName: 'McTesterson',
+    firstName: 'Testy',
+    ghinNumber: '1234567',
+    cachedGhinIndex: '0.0',
+    cachedGhinIndexDate: '2019-03-02',
+    driverClubHeadSpeed: '106.4',
+    dob: '2073-01-01',
+    address: '123 NW Abc St, Portland, OR 97229',
+    description: 'Recreational golfer'
+  }
+
+  describe('POST /api/v1/athletes', () => {
+    it('should create a new athlete', done => {
+      request(app)
+        .post('/api/v1/athletes')
+        .send(athlete)
+        .set('Accept', 'application/json')
+        .expect(201)
+        .end((err, res) => {
+          id = res.body.id
+          createdAt = res.body.createdAt
+          expect(res.body.lastName).to.equal('McTesterson')
+          done()
+        })
+    })
+  })
+
+  describe('POST /api/v1/athletes', () => {
```

Comment:
`describe` is used to group tests with similar topic but you have several describes for `POST /api/v1/athletes`

Reply:
That makes sense. Thanks. I was following a Mocha example and didn't realize I could do describe->it, it, it, etc. until you guy pointed it out. Will do the grouping to make the tests easier to follow.